libsplinter/src/admin/rest_api/mod.rs

```diff
@@ -24,6 +24,11 @@ use crate::admin::service::AdminService;
 use crate::admin::store::AdminServiceStore;
 use crate::rest_api::{Resource, RestResourceProvider};
 
+#[cfg(all(feature = "authorization", feature = "rest-api-actix"))]
```

Comment:
We should consider putting these in an rest_api::permissions:: module so we can protect them with a single cfg statement -- primarily because actix 3 stuff is going to start to hit the repo soon and this will get messy cfg-wise if we do it per-permission.